cloud/keystone_test.go

```diff
@@ -49,6 +49,29 @@ var _ = Describe("Keystone client", func() {
 		server.Close()
 	})
 
+	Describe("Tenant ID validation", func() {
+
+		BeforeEach(func() {
+			server.AppendHandlers(
+				ghttp.RespondWithJSONEncoded(201, getV3TokensScopedToTenantResponse()),
+				ghttp.RespondWithJSONEncoded(200, getV3TenantsResponse()),
+			)
+			setupV3Client()
+		})
+
+		It("Returns NotValidTenantIDErr when tenant id is not known", func() {
+			err := client.IsTenantIDValid("some-random-id")
```

Comment:
`Expect(client.IsTenantIDValid("some-random-id")).To(MatchError(NotValidTenantIDErr))`

Reply:
done